egs/fisher_swbd/s5/local/nnet3/run_ivector_common.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/bash
+
+. ./cmd.sh
+set -e
+stage=1
+train_stage=-10
+generate_alignments=true # false if doing ctc training
```

Comment:
Why? Don't we need the alignments for building tree? And since speed 
perturbed data is used, shouldn't we have a sp version of alignments? Or 
should I use the non-sp alignments for building tree?

在 2016/4/14 0:56, Vijayaditya Peddinti 写道:
>
> In egs/fisher_swbd/s5/local/nnet3/run_ivector_common.sh 
> <https://github.com/kaldi-asr/kaldi/pull/657#discussion_r59584490>:
>
> > @@ -0,0 +1,141 @@
> > +#!/bin/bash
> > +
> > +. ./cmd.sh
> > +set -e
> > +stage=1
> > +train_stage=-10
> > +generate_alignments=true # false if doing ctc training
>
> false if doing chain training.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/657/files/fc4140efa4727c4ad11cf689ae3faa26a9d0ffa0#r59584490>
>